src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,12 @@
+package lotto.domain
+
+import lotto.domain.generator.AutoLottoGenerator
+import lotto.domain.generator.LottoGenerator
+
+data class Lotto(private val generator: LottoGenerator = AutoLottoGenerator) {
+    private val numbers = generator.execute().sorted()
```

Comment:
`generator`가 꼭 필요할까요?

```suggestion
data class Lotto(private val numbers: List<Int>) {
    constructor(generator: LottoGenerator = AutoLottoGenerator) : this(
        generator.execute().sorted()
    )
```